src/core/server/saved_objects/migrationsv2/actions/index.ts

```diff
@@ -450,6 +456,15 @@ export const reindex = (
           index: sourceIndex,
           // Set reindex batch size
           size: BATCH_SIZE,
+          // Exclude saved object types
+          query: Option.fold<string[], QueryContainer | undefined>(
+            () => undefined,
+            (types) => ({
+              bool: {
+                must_not: types.map((type) => ({ term: { type } })),
```

Comment:
same NIT as before `terms` vs `term`? 🤷

Reply:
leaving as is since we don't gain anything from caching this search